cloudinit/debconf.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2022 Canonical Ltd.
```

Comment:
@aciba90 I don't think we really need to surface this top-level module yet, because our only consumer of this functionality in this PR is `cloudinit.config.cc_ubuntu_drivers` if we grow support for debconf use in other modules (like cc_apt*) then I could see us maybe needing this level of ImportError guarding in a top-level module definition. As it is, the only callers of this logic will be cc_ubuntu_drivers, so let's to the try/except ImportError dance in that module specifically. Since Ubuntu packaging will define a strict `Depends: python3-debconf` we won't be concerned about the absence of that import. Any distros will not otherwise need python3-debconf and so the ImportError handling can just pass in those cases.

Reply:
`debconf` import moved to `cloudinit.config.cc_ubuntu_drivers` and unit tested. Thanks.